### DISCUSSION TITLE
Documentation/conseil pour mise à jour manuelle du composant personnalisé Versatile Thermostat ?

### DISCUSSION THREAD

**Discussion by Lo-M83** [jmcollin78/versatile_thermostat#469] [Q&A]
Bonjour JM,

J'avoue humblement être bien débutant dans l'utilisation de Home Assistant, et j'ai découvert ce serveur domotique ainsi que le composant personnalisé Versatile Thermostat en même temps.
Du coup, j'ai équipé chacun des radiateurs à inertie de mes pièces à vivre de modules fil pilote Zigbee Leroy Merlin [Adeo SIN-4-FP-21_EQU] et les ai intégrés via ZHA (qui ne leur reconnait que le mode on/off, mais ça me suffit avec ton composant). J'ai ensuite créé tout un ensemble de VTherm (un par pièce, pour un contrôle fin des désidératas de chaque occupant, sans VTherm centralisé).
Cependant, au moment où j'ai découvert tout ce système de domotique, j'ai préféré une installation manuelle du composant personnalisé Versatile Thermostat, en suivant le Wiki de ton dépôt Git avec la version de l'époque : 5.4.0 (avec succès et sans aucune difficulté soit dit en passant, très simple à mettre en œuvre :thumb_up: ).
Maintenant en version 6.2.3, je me demande comment profiter des avancées et surtout comment mettre à jour manuellement ce composant personnalisé (et surtout, cela me fera-t-il perdre tous les VTherm déjà créés) ?
Si tu pouvais amender le Wiki (déjà très complet) avec un "How To" pour faire la MàJ manuelle du composant, avec peut-être les impacts sur une installation antérieure (perte ou conservation des VTherm déjà créés etc.), je te serais très reconnaissant !
Merci d'avance de ta réponse,

Laurent

**Comment by Lo-M83** [comment]
Bon, je m'autorépond : il m'a suffi d'écraser en FTP les fichiers dans HA (.../custom_component/versatile_thermostat/) avec ceux du même dossier récupéré sur le Git.
Je suis bien maintenant en version 6.2.3.

**Comment by jmcollin78** [comment]
L'installation et la configuration doivent se faire avec HACS et ne devraient pas être manuel. L'installation de HACS, si tu ne l'as pas déjà faite, est décrite ici : https://github.com/jmcollin78/versatile_thermostat/blob/6.2.3/README-fr.md#comment-installer-cet-incroyable-thermostat-versatile-
